backend/tasks/user.py

```diff
@@ -0,0 +1,79 @@
+from flask_restful import Resource, reqparse, inputs
+from flask import request
+from psycopg2.extras import RealDictCursor
+from pymysql.cursors import DictCursor
+import os
+from flask import make_response
+import json
+import logging
+from flask_login import login_required
+logger = logging.getLogger(__name__)
+
+
+class UserTask(Resource):
+
+    @login_required
+    def get(self, user):
+        if user is None or len(user) == 0:
+            return make_response(json.dumps({'message': 'Parameter \'user\' is required'}), 400)
+
+        try:
+            with request.db.cursor(cursor_factory=RealDictCursor) if os.environ.get('IS_LOCAL') else request.db.cursor(DictCursor) as cur:
+                cur.execute("""SELECT username, enabled FROM "user" WHERE username=%s""", (user,))
```

Comment:
I think table names and columns can't be quoted. Values need to be.
So `"user"` should be `user`. And `%s` needs to be quoted (`"%s"`)

Reply:
Unless it's different when using pymysql?

Reply:
%s shouldn't be quoted

https://www.psycopg.org/docs/usage.html#passing-parameters-to-sql-queries
https://pymysql.readthedocs.io/en/latest/user/examples.html

AFAIK `"user"` should be quoted as it is a keyword in PostgreSQL and MySQL.

Reply:
With backticks perhaps? That query as is throws a syntax error.

Reply:
Yeah mysql needs backticks and postgres needs quotes. They're mutually exclusive. I added a check on `IS_LOCAL` to determine which to use.